kubernetes/utils/create_from_yaml.py

```diff
@@ -14,60 +14,119 @@
 
 
 import re
-import sys
 from os import path
 
 import yaml
-from six import iteritems
 
 from kubernetes import client
 
 
-def create_from_yaml(k8s_client, yaml_file, verbose=False, **kwargs):
+def create_from_file(k8s_client, yaml_path, verbose=False, **kwargs):
```

Comment:
create_from_stream using a file like object would be more flexible than create from file path
That way you support anything file like at once and not only files on disk.

Reply:
I totally agree, but it seems that currently there will be no acceptance for that.